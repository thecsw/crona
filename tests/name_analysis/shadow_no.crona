a : int;

main : void(b : int) {
     a = b;
}