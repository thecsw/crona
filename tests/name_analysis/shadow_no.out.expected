a(int) : int;
main(int->void) : void(b(int) : int){
	a(int) = b(int);
}
